docs/guide/deploying.md

```diff
@@ -15,6 +15,18 @@ The following guides are based on some shared assumptions:
 }
 ```
 
+
+::: tip Use pages hosting provided by Git service
+
+Services such as GitHub, GitLab and other git services provide page hosting. (GitHub is used as an example below)
```

Comment:
Oxford comma should probably be added here (per [Vue Docs Writing Guide](https://github.com/vuejs/docs/blob/main/.github/contributing/writing-guide.md#grammar)).

Reply:
I just learned that the vue docs has such a writing guide, and I'm going to give it a good read.